types/layui-src/index.d.ts

```diff
@@ -0,0 +1,739 @@
+// Type definitions for layui-src 2.5
+// Project: https://github.com/javabitar/layui
```

Comment:
Shouldn't this be rather thsi one?
```suggestion
// Project: https://github.com/sentsin/layui
```